apps/api/src/app/bridge/usecases/preview-step/preview-step.usecase.ts

```diff
@@ -37,10 +39,29 @@ export class PreviewStep {
       controls: command.controls || {},
       payload: command.payload || {},
       state: command.state || [],
-      subscriber: command.subscriber || {},
+      subscriber: this.mapSubscriberResponseToSubscriber(command.subscriber),
       stepId: command.stepId,
       workflowId: command.workflowId,
       action: PostActionEnum.PREVIEW,
     };
   }
+
+  private mapSubscriberResponseToSubscriber(
+    subscriberResponse?: Partial<SubscriberResponseDto> | undefined
+  ): Subscriber {
+    if (!subscriberResponse) {
+      return {
+        id: '',
+        subscriberId: '',
+      };
+    }
+
+    return {
+      id: subscriberResponse._id || subscriberResponse.subscriberId || '',
```

Comment:
_id is always present and subscriberId must never be returned in lieu of _id.
```suggestion
      id: subscriberResponse._id,
```

Reply:
it's not returning it, this code builds the preview requiest to the hosted bridge, I made the fix as you suggested